pipelines/batch/src/main/java/com/google/fhir/analytics/ConvertResourceFn.java

```diff
@@ -97,7 +100,7 @@ public class ConvertResourceFn extends FetchSearchPageFn<HapiRowDescriptor> {
 
   public void writeResource(HapiRowDescriptor element)
       throws IOException, ParseException, SQLException, ViewApplicationException, ProfileException {
-    String resourceId = element.resourceId();
+    String fhirId = element.fhirId();
```

Comment:
**question:** I wonder how they'll see this change. May be "breaking" for them.

Reply:
Yeah, might need a "compatibility" option but it didn't really make sense

Reply:
Yeah, we can wait for them to complain on the PR to their code base